libopenage/renderer/opengl/buffer.cpp

```diff
@@ -50,4 +62,8 @@ void GlBuffer::bind(GLenum target) const {
 	glBindBuffer(target, *this->handle);
 }
 
+void GlBuffer::unbind(GLenum target) const {
```

Comment:
Like for the texture, unbinding is not necessary.

Reply:
A method for doing it wouldn't hurt☺

Reply:
For maintainability, it's better to have less than more code, so if something is unused, better to remove it.